server/repository/src/db_diesel/changelog/README.md

```diff
@@ -0,0 +1,43 @@
+# Changelog
+
+A number of operations rely on processing a `queue` of database changes.
+These operations include pushing records to central server during synchronisation, processing transfer records, processing messaging, aggregation, interoperability synchronisation, etc.. 
+
+Generic mechanism for recording and querying these database changes was implemented in the form of the `changelog` table.
+
+## Triggers
+
+Database triggers insert new entries into changelog table with:
+
+* cursor <- sequential number for order of database operation
+* record_id 
+* table_name
+* store_id
```

Comment:
```suggestion
* store_id <- the store that `owns` the record
* name_id <- the name_id of the `destination` entity (often also a store)
```

Not sure if this needs to be here, but might help someone understand the distinction. I'm guessing there might be exceptions to these rules?

Reply:
I've change `(often also a store)` to just `(can also be a store)` 5707429b6383efb1ddbd03b77b08316453bfc2fe